checkInclusion.m

```diff
@@ -0,0 +1,79 @@
+function [stopflag] = checkInclusion(location, grid, data);
```

Comment:
This seems unnecessarily complicated. Don't we just want to check whether the value at some state x (location) is less than 0?